Sources/Common/Background Queue/QueueRunRequest.swift

```diff
@@ -135,7 +135,14 @@ public class CioQueueRunRequest: QueueRunRequest {
                         self.logger.info("queue is quitting early because all HTTP requests are paused.")
 
                         doneRunning()
-                        break
```

Comment:
It was very confusing, took me some time to grasp it as well. 

So there is a loop and inside it, there is a switch statement, `break` statement caused confusion so I opted for exhaustive approach.